src/aave-v2/SupplyVaultBase.sol

```diff
@@ -76,6 +79,10 @@ abstract contract SupplyVaultBase is ERC4626UpgradeableSafe, OwnableUpgradeable
 
     /// EXTERNAL ///
 
+    /// @dev Token rescue function.
```

Comment:
This is not a token rescue function but the required function to retrieve MORPHO rewards